### PR TITLE
add Seed-1.6-embedding model results

### PR DESCRIPTION
**Pull request by QuanYuhan**
### Checklist

- [x] My model has a model sheet, report or similar
- [] My model has a reference implementation in [`mteb/models/`](https://github.com/embeddings-benchmark/mteb/tree/main/mteb/models) this can be as an API. Instruction on how to add a model can be found [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/reproducible_workflow.md) 
  - [x] No, but there is an existing PR https://github.com/embeddings-benchmark/mteb/pull/2841#issue-3165894062
- [x] The results submitted is obtained using the reference implementation
- [x] My model is available, either as a publicly accessible API or publicly on e.g., Huggingface
- [x] I *solemnly swear* that for all results submitted I have not on the evaluation dataset including training splits. If I have I have disclosed it clearly.

### CODE REVIEW

**Comment by QuanYuhan** [comment]
@KennethEnevoldsen I noticed that a "review of implementation" tag has been added to this PR. May I ask how long this usually takes? What other processes are probably needed so that these results can be merged. If you need to add any extra information or have any questions, please let me know at any time.

**Comment by KennethEnevoldsen** [comment]
@QuanYuhan, it simply means that we are waiting for a review of the implementation (change the name to match).

I just merged the implementation so will take a look at this now

**Comment by KennethEnevoldsen** [comment]
Potentially problematic:
- TwentyNewsgroupsClustering.v2 
- AmazonCounterfactualClassification
- ClimateFEVERHardNegatives
- TweetSentimentExtractionClassification
- MassiveScenarioClassification

@QuanYuhan, some of these scores look like the data might have been included during training (some of them are comparable to models trained on the training set). Can you double-check these?

**Comment by QuanYuhan** [comment]
> Potentially problematic:
> 
> * TwentyNewsgroupsClustering.v2
> * AmazonCounterfactualClassification
> * ClimateFEVERHardNegatives
> * TweetSentimentExtractionClassification
> * MassiveScenarioClassification
> 
> @QuanYuhan, some of these scores look like the data might have been included during training (some of them are comparable to models trained on the training set). Can you double-check these?

@KennethEnevoldsen  Thank you for your reminder.

After a careful check, we found that there were omissions in the training data. For the 5 datasets you've mentioned, we
didn't use ClimateFEVERHardNegatives training sets, for other datasets, we used the training set. Meanwhile, we have
supplemented some other training set information.

We have submitted a PR(https://github.com/embeddings-benchmark/mteb/pull/2857) to update the model information. If you have any other questions, feel free to contact me at any time.

**Comment by QuanYuhan** [comment]
@KennethEnevoldsen  Please have a look. Can the results be merged? Thank you.

**Review by Samoed — APPROVED** [review]
